src/main/java/frc/robot/RobotContainer.java

```diff
@@ -98,13 +113,14 @@ private void configureButtonBindings() {
     fourBarClimb.whenHeld(new ClimbRoutine(redArm, yellowArm, tiltArm));
   }
 
+
   /**
    * Use this to pass the autonomous command to the main {@link Robot} class.
    *
    * @return the command to run in autonomous
    */
-  public Command getAutonomousCommand() {
+  public Object getAutonomousCommand() {
```

Comment:
As a general rule you should not return `Object` from a function. I think leaving that as `Command` was correct.